src/test_utils/contract_call.rs

```diff
@@ -0,0 +1,171 @@
+use crate::parameters::{FunctionCallArgs, SubmitResult};
+use crate::prelude::Address;
+use crate::test_utils::{solidity, AuroraRunner, Signer};
+use crate::transaction::EthTransaction;
+use borsh::BorshSerialize;
+use std::convert::TryInto;
+
+pub(crate) struct TesterConstructor(pub solidity::ContractConstructor);
```

Comment:
All the naming here is very generic; it's a little hard for me to know exactly what this is testing. I don't necessarily think any of these names need to change if the module name changes though. For example `exit_precompiles.rs`.